code/drasil-metadata/lib/Data/Drasil/DrasilMetaCall.hs

```diff
@@ -0,0 +1,10 @@
+{-#LANGUAGE TemplateHaskell#-}
+module Data.Drasil.DrasilMetaCall (drasilMeta, DrasilMeta(..))where
+
+import Data.Drasil.DrasilMeta (drasilMetaCfg, DrasilMeta(..))
+import Data.Maybe (fromMaybe)
+
+
+-- | Reads drasilMeta at compile-time
+drasilMeta :: DrasilMeta
+drasilMeta = $drasilMetaCfg
```

Comment:
Unfortunately, this has to be in another file or else GHC causes an error. However, I see that we don't have the 'normal' 1-file-exposed setup for `drasil-metadata`. Maybe we should create a `Metadata.Drasil` and have this there? Also, should we rename the module names to `Metadata.Drasil`, @JacquesCarette ?

Reply:
We shouldn't have a `Metadata.Drasil` with "everything" in it. Renaming those modules to the suggested namespace is a good diea.